### PR TITLE
Correctly enforce upper case title IDs

### PR DESCRIPTION
**Pull request by j-selby**
This PR changes the behaviour of the validator to enforce that all title IDs are uppercase, ensuring consistency. This also fixes games which currently fail this validation.

### CODE REVIEW

File: validation/app.js [hunk 1]
```diff
@@ -200,14 +200,14 @@ function validateTOML(path) {
         section.forEach(release => {
             validateContents(release, "title", field => {
                 if (field.length !== 16) {
-                    validationError(`Release #${i + 1}: Game title ID has an invalid length`);
```
Comment (TheKoopaKingdom) [inline validation/app.js]: What does this have to do with this PR?

Comment (j-selby) [reply validation/app.js]: Nothing, really, but we no longer have a index to compare against (see the forEach method call instead) - just got bunched in here. Happy to rebase, but it still is broken code.
